panda/src/putil/bamReader.cxx

```diff
@@ -133,6 +133,10 @@ init() {
     _file_stdfloat_double = scan.get_bool();
   }
 
+  if(_file_minor >= 46) {
+    _tex_compression_level = (BamTexCompressionLv)scan.get_uint8();
```

Comment:
We should throw a meaningful error if the compression format/level requested is not supported.